projects/demo/src/environments/environment.ts

```diff
@@ -0,0 +1,16 @@
+// This file can be replaced during build by using the `fileReplacements` array.
+// `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
```

Comment:
suggestion: The `--prod` parameter is deprecated in Angular 12 and this generated comment is changed to reflect that.
```suggestion
// `ng build` replaces `environment.ts` with `environment.prod.ts`.
```